modules/administration/pages/icinga.adoc

```diff
@@ -128,7 +137,7 @@ vi /etc/icinga/ido2db.cfg
 [IMPORTANT]
 .Open Firewall Port
 ====
-Allow port `5432` through your firewall or you will not be able to access the WebGUI.
+Allow port `80` through your firewall or you will not be able to access the WebGUI.
```

Comment:
Either I am quite wrong, or 5432 was incorrect. 5432 is for PostgreSQL, that it's to be used by ido2b when it receives data from nrpe.

Reply:
That is correct, thank you.